instrumentation/jms-1.1/javaagent/src/test/groovy/SpringTemplateJms1Test.groovy

```diff
@@ -97,6 +97,17 @@ class SpringTemplateJms1Test extends AgentInstrumentationSpecification {
     expect:
     receivedMessage.text == "responded!"
     assertTraces(4) {
+      sortTraces {
+        def expectedOrder = ["SpringTemplateJms1 receive",
+                             "SpringTemplateJms1 send",
```

Comment:
Consider changing to 

```suggestion
        def expectedOrder = ["$destinationName receive",
                             "$destinationName send",
```

to avoid hard-coding the test param

Reply:
thanks, changed.